Jenkinsfile

```diff
@@ -128,25 +128,22 @@ pipeline {
                                 }
                             }
                         }
-                        stage('Rocky 8') {
+                        stage('RHEL') {
                             agent {
                                 node {
-                                    label 'pacur-agent-rocky-8-v1'
+                                    label 'yap-agent-rocky-8-v2'
                                 }
                             }
                             steps {
                                 dir('/tmp/staging'){
                                     unstash 'binaries'
                                 }
-                                sh 'sudo pacur build rocky-8 /tmp/staging/'
-                                dir('artifacts/') {
-                                    sh 'echo carbonio-files* | sed -E "s#(carbonio-files-ce-[0-9.]*).*#\\0 \\1.x86_64.rpm#" | xargs sudo mv'
```

Comment:
Is this `sed` trick handled by yap itself now?

Reply:
this trick was due to make the artifactory matches work. Using the correct mask and the correct filename templating, make these few lines unneeded